Resources/Prototypes/Entities/Objects/Specific/Medical/morgue.yml

```diff
@@ -64,6 +64,11 @@
   components:
   - type: Foldable
     folded: true
+  - type: Tag
+    tags:
+    - BodyBag
+  - type: Item
+    size: 6
```

Comment:
These should be on the base copy. The only difference is this is pre-folded for reasons.

Reply:
If I do this this lets you pick up and drop unfolded body bags, which seems to break. Folded ones work fine

Reply:
You moved the item over, but still need to move the tag over.